src/bacnet/basic/bbmd6/h_bbmd6.c

```diff
@@ -32,7 +32,7 @@
 #endif
 #endif
 
-static bool BVLC6_Debug;
+static bool BVLC6_Debug = false;
```

Comment:
static file scope gets set to zeros as part of BSS per C compiler rules, so no need to set to false again.